.github/workflows/debug-build.yml

```
@@ -124,6 +131,38 @@ jobs:
           name: stacks-wallet-${{ matrix.stx_network }}-${{ matrix.NPM_COMMAND }}
           path: ${{ matrix.UPLOAD_ASSETS }}
 
+  asset-signatures:
+    runs-on: ubuntu-latest
+    needs:
+      - debug-build
+
+    steps:
+      - name: Download binaries
+        uses: actions/download-artifact@v2
+        with:
+          path: downloads
+
+      - run: ls -R downloads
+
+      - name: Flatten assets to root dir
+        run: find downloads -type f -exec mv '{}' . \; && rm -rf downloads
+
+      - run: ls | xargs -I{} shasum -a 256 {} | grep -v shasum.txt | tee -a shasum.txt
+
+      - name: Import GPG key
+        id: import_gpg_key
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+
+      - run: gpg --armor --output shasum.txt.asc --clearsign shasum.txt
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: shasum.txt.asc
+          path: shasum.txt.asc
```

Can this be simplified to reduce the number of steps here? I'm thinking something like:

```suggestion
      - name: Download binaries
        uses: actions/download-artifact@v2
        with:
          path: downloads

      - name: Import GPG key
        id: import_gpg_key
        uses: crazy-max/ghaction-import-gpg@v3
        with:
          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
          passphrase: ${{ secrets.GPG_PASSPHRASE }}

      - name: Generate binary shasums
        run: |
          find downloads -type f -exec mv '{}' . \; && rm -rf downloads
          ls | xargs -I{} shasum -a 256 {} | grep -v shasum.txt | tee -a shasum.txt
          gpg --armor --output shasum.txt.asc --clearsign shasum.txt

      - name: Upload binary shasums
        uses: actions/upload-artifact@v2
        with:
          name: shasum.txt.asc
          path: shasum.txt.asc
```